synctl

```diff
@@ -2745,8 +3007,20 @@ def main():
                     print("Synthetic type only support 0 1 2 3 4", syn_type_t)
 
             if get_args.id is None:
+                if get_args.filter is not None:
+                    split_string = get_args.filter.split('=')
+                    if split_string[0] == "locationId":
+                        filtered_payload = syn_instance.retrieve_synthetic_test_by_location(split_string[1])
+                    elif split_string[0] == "applicationId":
+                        filtered_payload = syn_instance.retrieve_synthetic_test_by_application(split_string[1])
+                    syn_instance.print_tests_by_filter(filtered_payload, split_string[0])
```

Comment:
if filter are not `locationId` and `applicationId` print an error message and exit.